WarzoneMaps/solar/invalidsmallsolar.map

```diff
@@ -0,0 +1,28 @@
+; map: canada.map
```

Comment:
we will have to either remove that (its referencing another map) or put something else. In any case, it shouldnt affect the reading of the map

Reply:
We can leave it since it is not read by anything in the code